src/main/resources/liquibase/changelog/20190808100311_changelog.sql

```diff
@@ -0,0 +1,46 @@
+INSERT INTO newEbodac.field_config (id, base, display_name, entity, format, hidden, name, required, field_type, sort_order, create_date, update_date, related_entity, related_field, editable, filterable) VALUES
+    ('83e52443-3a96-474c-8104-ce7c2cd68ea4', 1, 'ID', 'VACCINEE', '', 0, 'id', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('1caa5c25-0860-4a70-b37c-66c64776c064', 1, 'Vaccinee', 'VACCINEE', '', 0, 'vaccineeId', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('ca620d2e-cf22-484f-8ab5-49f3d9bbb23a', 1, 'Name', 'VACCINEE', '', 0, 'name', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('96ca07dc-5ce5-4664-a9e1-51a94dbe55b9', 1, 'Gender', 'VACCINEE', 'MALE:Male,FEMALE:Female,UNKNOWN:Unknown', 0, 'gender', 0, 'ENUM', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', '', '', 0, 0),
+    ('94050a63-23e8-41f8-8ab3-2b94f912c948', 1, 'Age', 'VACCINEE', '', 0, 'age', 0, 'INTEGER', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('9eecadf9-21f9-4f04-bebe-e96908201a45', 1, 'Year of birth', 'VACCINEE', '', 0, 'yearOfBirth', 0, 'INTEGER', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('a05292d0-430d-4403-9b9f-e9f3f831a0ca', 1, 'Phone Number', 'VACCINEE', '', 0, 'phoneNumber', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('cd98a8e9-04bf-41eb-b49a-9557089a3231', 1, 'Alternate Phone Number', 'VACCINEE', '', 0, 'alternatePhoneNumber', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('942c7144-1c09-4ab7-80e8-0f52454234d1', 1, 'Preferred language name', 'VACCINEE', '', 0, 'preferredLanguageName', 0, 'TEXT', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', null, null, 0, 0),
+    ('8ea9c4fe-5e6f-4685-9894-2971bb1ba990', 1, 'Preferred language id', 'VACCINEE', '', 0, 'preferredLanguageId', 0, 'RELATION', 0, '2019-08-08 15:28:20', '2019-08-08 15:28:20', 'LANGUAGE', 'preferredLanguage', 0, 0),
```

Comment:
change the display name to "Preferred language", related field should be "name", because we want to display the language name in this field (in the table or the select)